x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/CcrSettings.java

```diff
@@ -28,6 +29,12 @@ private CcrSettings() {
     public static final Setting<Boolean> CCR_FOLLOWING_INDEX_SETTING =
             Setting.boolSetting("index.xpack.ccr.following_index", false, Property.IndexScope, Property.InternalIndex);
 
+    /**
+     * Dynamic node setting for specifying the wait_for_timeout that the auto follow coordinator should be using.
+     */
+    public static final Setting<TimeValue> CCR_AUTO_FOLLOW_COORDINATOR_WAIT_FOR_TIMEOUT = Setting.timeSetting(
+        "ccr.auto_follow_coordinator.wait_for_timeout", TimeValue.timeValueSeconds(60), Property.NodeScope, Property.Dynamic);
```

Comment:
Should we come up with another name which indicates that this is a timeout setting for wait_for_metadata_version?

Reply:
Is `wait_for_metadata_timeout` a better name?

Reply:
Yeah, it's better :). I think we should also drop `coordinator` in the setting name.